data/data/manifests/openshift/cluster-authentication-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: authentications.config.openshift.io
+spec:
```

Comment:
Set `spec.subresources.status` to the empty object for both resources.

Reply:
done